plugins/woo-ai/includes/class-woo-ai-settings.php

```diff
@@ -0,0 +1,220 @@
+	 * Add UI related hooks.
+	 */
+	public function add_ui() {
+		add_filter( 'woocommerce_get_settings_advanced', array( $this, 'add_woo_ai_settings' ), 10, 2 );
+	}
+
+	/**
+	 * Save settings.
+	 */
+	public function action_save_woo_ai_settings_tab() {
+		WC_Admin_Settings::save_fields(
+			$this->get_woo_ai_settings()
+		);
+	}
+
+	/**
+	 * Add sanitization hooks.
+	 */
+	public function add_sanitization_hooks() {
+		$settings = $this->get_woo_ai_settings();
+
+		foreach ( $settings as $setting ) {
+			if ( in_array( $setting['type'], array( 'text', 'textarea' ), true ) ) {
+				add_filter( 'woocommerce_admin_settings_sanitize_option_' . $setting['id'], array( $this, 'strip_tags_field_value' ) );
+			}
+		}
+	}
+
+	/**
+	 * Sanitize field value.
+	 *
+	 * @param string $raw_value The current section.
+	 */
+	public function strip_tags_field_value( $raw_value ) {
+		return wp_strip_all_tags( $raw_value ?? '' );
+	}
+
+	/**
+	 * Add settings to the AI section.
+	 *
+	 * @param array  $settings The original settings array.
+	 * @param string $current_section The current section.
+	 */
+	public function add_woo_ai_settings( $settings = array(), $current_section = null ) {
+		if ( 'features' === $current_section ) {
+			$settings = array_merge(
+				$this->get_woo_ai_settings(),
+				$settings
+			);
+		}
+
+		return $settings;
+	}
+
+	/**
+	 * Return array describing new settings.
+	 *
+	 * @return array Describing new AI settings.
+	 */
+	public function get_woo_ai_settings() {
+
+		// These are made available in the DOM and actually toggled client-side.
+		$tone_desc = array(
+			'informal'     => __( 'Relaxed and friendly, as if you were having a conversation with a friend.', 'woocommerce' ),
+			'humorous'     => __( 'Engage customers with a light-hearted and fun communication style.', 'woocommerce' ),
+			'neutral'      => __( 'A balanced and impartial tone that uses casual expressions without slang.', 'woocommerce' ),
+			'youthful'     => __( 'Bring energy and enthusiasm to your store with a friendly and cheeky tone.', 'woocommerce' ),
+			'formal'       => __( 'Direct yet respectful, a formal tone sounds serious and professional.', 'woocommerce' ),
+			'motivational' => __( 'Passionate and inspiring, engage users with this upbeat tone.', 'woocommerce' ),
+		);
+
+		$markup_str = array_reduce(
+			array_keys( $tone_desc ),
+			function( $acc, $key ) use ( $tone_desc ) {
+				$value = $tone_desc[ $key ];
+				return $acc . '<span class="woo-ai-settings-tone-option" data-option="' . htmlspecialchars( $key ) . '">' . htmlspecialchars( $value ) . '</span>';
+			},
+			''
+		);
+
+		return array(
+			array(
+				'id'    => 'woo_ai_title',
+				'type'  => 'title',
+				'title' => __( 'Artificial Intelligence', 'woocommerce' ),
+				'desc'  => __( "Save time by automating mundane parts of store management. This information will make AI-generated content, visuals, and setting more aligned with your store's goals and identity.", 'woocommerce' ),
```

Comment:
Just noticed this should say "settings", not "setting".

Reply:
Thanks for seeing this - addressed as part of https://github.com/woocommerce/woocommerce/pull/39565